index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>HomePage - Fourth International Conference on Science and Technology</title>
+</head>
+
+<body class="body">
+
+    <div class="wrapper">
```

Comment:
- [ ] [OPTIONAL] kindly reduce the use of the `div` tag in your code, instead use semantic HTML tags like `section` and `article`, this will improve the readability of your code and improve SEO